.stylelintrc

```diff
@@ -86,15 +81,13 @@
     "selector-list-comma-space-after": "always-single-line",
     "selector-list-comma-space-before": "never",
     "selector-max-specificity": "0,4,0",
-    "selector-no-id": true,
-    "selector-no-type": true,
-    "selector-no-universal": true,
```

Comment:
Except the `no-type`, I think it makes sense to keep this.

Reply:
I agree